MainRepository/Program.cs

```diff
@@ -1,7 +1,38 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿//Chocolate Wrappers
+while (true) {

```

Comment:
Use at least 1 keyword/technique: in, out, ref, optional parameters, multiple return values.